app/models/room.rb

```diff
@@ -4,7 +4,11 @@ class Room < ApplicationRecord
   belongs_to :building
   has_many :events, dependent: nil
   has_and_belongs_to_many :people
+  has_many :favourite_rooms
```

Comment:
Is this necessary? I think `has many through` should suffice?

Reply:
I don't know, didn't write that code, but when I remove it everything crashes 🤷